index.md

```diff
@@ -9,55 +9,192 @@ iOS and Android from a single codebase.
+on different platforms, and we embrace differences like scrolling behavior, 
+overscroll behaviors, and more. 
 
- - [Set up your machine](/setup/) with Flutter and iOS/Android tools.
- - [Get started](getting-started) to run your first app on Android and iOS.
- - [A tour of the Flutter framework](widgets-intro)
- - [Bootstrap into Dart](/bootstrap-into-dart/) if you are new to the language
- for developing Flutter apps.
+![Shrine, iOS left and Android right](/images/whatisflutter/shrine_both.png)
 
-## Developer Guides
+This is an online marketplace app from the [Gallery](https://github.com/flutter/flutter/tree/master/examples/flutter_gallery/lib/demo), 
+a showcase of Flutter apps to download and run on a phone. Shrine has 
+high-quality scrolling images, interactive cards, buttons and dropdown lists, 
+and a shopping cart page. Run Shrine on an iPhone or Android phone; the UI 
+appearance and behavior are equivalent. Feel free to browse through the source 
+code and other sample apps [here](https://github.com/flutter/flutter/tree/master/examples) 
+in our github repository.
+
+No mobile development experience is required to get started. Apps are written 
+in [Dart](https://dartlang.org), which will look familiar if you've used a 
+language like Java or JavaScript. While experience with object-oriented languages 
+is definitely helpful, even non-programmers have made Flutter apps!
 
-See these topics to understand how the Flutter framework works.
+## Why use Flutter?
 
- - [Widgets overview](widgets)
- - [Gestures in Flutter](gestures)
- - [Animations in Flutter](animations)
- - [Using custom fonts](custom-fonts)
- - [Layered design (video)](https://www.youtube.com/watch?v=dkyY9WCGMi0)
- - [Rendering pipeline (video)](https://www.youtube.com/watch?v=UUfXWzp0-DU)
- - [Dealing with box constraints in Flutter](layout)
+What are some advantages of Flutter? It helps you:
 
-## How-Tos
-
-Learn how to accomplish specific development tasks with Flutter.
-
- - [Building Layouts in Flutter](/tutorials/layout/)
- - [Adding Assets and Images](assets-and-images)
- - [Testing Flutter Apps](testing)
- - [Debugging Flutter Apps](debugging)
- - [Developing apps in the IntelliJ IDE](intellij-ide)
- - [Formatting Flutter code](formatting)
- - [Upgrading Flutter](upgrading)
- - [Accessing Platform and Third-Party Services](platform-services)
- - [Reading and Writing Files](reading-writing-files)
- - [Handling Text Input](text-input)
-
-## Helpful Resources
-
-More resources for Flutter developers.
-
- - [Source on GitHub](https://github.com/flutter/flutter)
- - [Architecture diagram](https://docs.google.com/presentation/d/1cw7A4HbvM_Abv320rVgPVGiUP2msVs7tfGbkgdrTy0I/edit?usp=sharing)
- - [Issue tracker](https://github.com/flutter/flutter/issues)
- - [Writing useful bug reports](bug-reports)
- - [FAQ](faq)
- - [Design principles](design-principles)
-
-## Support
+*   Be highly productive
```

Comment:
Mentioned it in a lower bullet. It is a cool feature...but we're working hard to keep this page high-level.